packages/tools/src/utilities/contours/PointsArray.ts

```diff
@@ -14,152 +17,189 @@ export type PolyDataPointConfiguration = {
  * transferring them amongst systems and is planned to have more methods added
  * for generic manipulation of data.
  */
-export abstract class PointsArray<T> {
+export class PointsArray<T> {
```

Comment:
I believe it would be beneficial to include this in the core, considering that we already have voxelManager there. Should we refer to this as a manager as well? After all, it serves a similar purpose.